packages/subgraph/tests/dao/dao_v1_3_0.test.ts

```diff
@@ -250,7 +248,7 @@ describe('handleExecuted', () => {
 
   describe('ERC20 action', () => {
     beforeAll(() => {
-      createTokenCalls(DAO_TOKEN_ADDRESS, 'name', 'symbol', '6', '10');
+      createERC20TokenCalls(DAO_TOKEN_ADDRESS, '10', 'name', 'symbol', '6');
```

Comment:
Would be useful to provide some context on '10' and '6' by replacing the magic numbers with a constant at the top of the file